node/flatpak-node-generator.py

```diff
@@ -1056,7 +1059,7 @@ async def generate_special_sources(self, package: Package) -> None:
             self._handle_electron_builder(package)
         elif package.name == 'gulp-atom-electron':
             self._handle_gulp_atom_electron(package)
-        elif package.name == 'dugite':
+        elif package.name in ['dugite', '@shiftkey/dugite']:
```

Comment:
minor nit: use a set comparison, it's slightly faster (but also a bit clearer, since sets are often used and thus associated with membership tests)

Reply:
Makes sense, done.